╒═══════════════════╤════════╕
│ run config name   │        │
╞═══════════════════╪════════╡
│ CONFIG_NAME       │ debug  │
├───────────────────┼────────┤
│ device            │ cuda   │
├───────────────────┼────────┤
│ epochs            │ 20     │
├───────────────────┼────────┤
│ learning rate     │ 0.0001 │
├───────────────────┼────────┤
│ batch size        │ 8      │
╘═══════════════════╧════════╛
Dataset with 14041 samples across 1 sites.
Starting epoch 1/20.
C:\Users\shafner\AppData\Local\Continuum\anaconda3\envs\local\lib\site-packages\torch\nn\modules\loss.py:528: UserWarning: Using a target size (torch.Size([8])) that is different to the input size (torch.Size([8, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
Logging step 100 (epoch 0.06).
RMSE training 180.358
RMSE test 180.777
Logging step 200 (epoch 0.11).
RMSE training 176.986
RMSE test 180.603
Logging step 300 (epoch 0.17).
RMSE training 164.268
RMSE test 169.475
Logging step 400 (epoch 0.23).
RMSE training 169.813
RMSE test 168.481
Logging step 500 (epoch 0.28).
RMSE training 165.853
RMSE test 179.184
Logging step 600 (epoch 0.34).
RMSE training 172.142
RMSE test 163.349
Logging step 700 (epoch 0.40).
RMSE training 172.688
RMSE test 178.111
Logging step 800 (epoch 0.46).
RMSE training 163.705
RMSE test 175.740
Logging step 900 (epoch 0.51).
RMSE training 171.014
RMSE test 171.017
Logging step 1000 (epoch 0.57).
RMSE training 173.460
RMSE test 172.083
Logging step 1100 (epoch 0.63).
RMSE training 184.283
RMSE test 181.228
Logging step 1200 (epoch 0.68).
RMSE training 175.924
RMSE test 174.839
Logging step 1300 (epoch 0.74).
RMSE training 174.664
RMSE test 181.469
Logging step 1400 (epoch 0.80).
